packages/server/src/plugin/landingPage/default/index.ts

```diff
@@ -68,7 +70,7 @@ const getNonEmbeddedLandingPageHTML = (
   <p>The full landing page cannot be loaded; it appears that you might be offline.</p>
 </div>
 <script>window.landingPage = ${encodedConfig};</script>
-<script src="https://apollo-server-landing-page.cdn.apollographql.com/${version}/static/js/main.js"></script>`;
+<script src="https://apollo-server-landing-page.cdn.apollographql.com/${cdnVersion}/static/js/main.js?referrer=${apolloServerVersion}"></script>`;
```

Comment:
Is `referrer` a thing that already exists on the cdn side in some way or is the query param name still up for debate? I have a very minor concern that this is overloading a term which is the name of a header that's used for URLs (though I see the parallel and it makes sense, so it might be fine).

Reply:
This is just something we are going to look for in our data logs of http requests. Any other naming suggestions?

Reply:
No strong opinion, maybe `runtime`, `serverRuntime`, or something along those lines?

Things that maybe affect my consideration that I don't have the answer to:
Will the router send this as well? Is there any benefit to us or them if a 3rd party included their runtime if they were using sandbox?

Reply:
yes the router will send us this, I could see something else that embeds the explorer sending us this too. I'll go with `runtime`